packages/x-components/src/x-modules/facets/components/filters/all-filter.vue

```diff
@@ -3,9 +3,10 @@
     class="x-filter x-all-filter"
     data-test="all-filter"
     role="checkbox"
-    :aria-checked="facet.label.toString()"
+    :aria-checked="isSelected"
     :events="clickEvent"
     :class="cssClasses"
+    :aria-labelledby="facet.label"
```

Comment:
As this value will be mapped into an HTML ID from another part of the application, should if have a prefix/sufix to "join" both elements without any other component using the same ID? Maybe something like:
```suggestion
    :aria-labelledby="`all-filter-${facet.label}`"
```